network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -835,7 +835,7 @@ class SoftwareProductPage(ProductPage):
     # NullBooleanField is deprecated as of Django 3.1.
     # We're using it here primarily for a data migration, but we should
     # move to BooleanField as soon as it's safe to do so with the content we have
-    easy_to_learn_and_use = models.NullBooleanField(
+    easy_to_learn_and_use = models.BooleanField(
```

Comment:
NullBooleanField is deprecated and being removed in Django 4. There was a warning in the console when using it, so I changed this field type. BooleanField(null=True) is supported.